cms/models.py

```diff
@@ -244,6 +254,25 @@ class Meta:
         help_text="Image that will be used where the course is featured or linked.",
     )
 
+    def _get_player_config(self):
```

Comment:
Nitpick: this could be a `@property`. You could just call it `def video_player_config`